neptune/alpha/leaderboard.py

```diff
@@ -0,0 +1,153 @@
+                _type = attr.type
+                if _type == AttributeType.FILE:
+                    self._backend.download_file(self._id, parse_path(path), destination)
+                    return
+                raise MetadataInconsistency
+        raise AttributeError
+
+    def download_file_set_attribute(self, path: str, destination: Optional[str]):
+        for attr in self._attributes:
+            if attr.path == path:
+                _type = attr.type
+                if _type == AttributeType.FILE_SET:
+                    self._backend.download_file_set(self._id, parse_path(path), destination)
+                    return
+                raise MetadataInconsistency
+        raise AttributeError
+
+
+class LeaderboardHandler:
+
+    def __init__(self, experiment: LeaderboardExperiment, path: str):
+        self._experiment = experiment
+        self._path = path
+
+    def __getitem__(self, path: str) -> 'LeaderboardHandler':
+        return LeaderboardHandler(self._experiment, join_paths(self._path, path))
+
+    def get(self):
+        return self._experiment.get_attribute(self._path)
+
+    def download(self, destination: Optional[str]):
+        return self._experiment.download_file_attribute(self._path, destination)
+
+    def download_zip(self, destination: Optional[str]):
+        return self._experiment.download_file_set_attribute(self._path, destination)
+
+
+class Leaderboard:
+
+    def __init__(self, backend: HostedNeptuneBackend, entries: List[LeaderboardEntry]):
+        self._backend = backend
+        self._entries = entries
+
+    def as_experiments(self) -> List[LeaderboardExperiment]:
+        return [LeaderboardExperiment(self._backend, e.id, e.attributes) for e in self._entries]
+
+    def as_pandas(self):
+        # pylint:disable=import-outside-toplevel
+        import pandas as pd
+
+        def make_attribute_value(attribute: AttributeWithProperties) -> Optional[Union[str, float, datetime]]:
+            _type = attribute.type
+            _properties = attribute.properties
+            if _type == AttributeType.EXPERIMENT_STATE:
+                return _properties.value
+            if _type == AttributeType.FLOAT or _type == AttributeType.STRING or _type == AttributeType.DATETIME:
+                return _properties.value
+            if _type == AttributeType.FLOAT_SERIES or _type == AttributeType.STRING_SERIES:
+                return _properties.last
+            if _type == AttributeType.IMAGE_SERIES:
+                return None
+            if _type == AttributeType.FILE or _type == AttributeType.FILE_SET:
+                return None
+            if _type == AttributeType.STRING_SET:
+                return ",".join(_properties.values)
+            if _type == AttributeType.GIT_REF:
+                return _properties.commit.commitId
+            if _type == AttributeType.NOTEBOOK_REF:
+                return _properties.notebookName
+            raise AttributeError
```

Comment:
InternalClientError or None.
You can skip images and files cases if you use None.